README.textile

```diff
@@ -69,6 +69,10 @@ the problem. Also see "this FAQ":https://web.archive.org/web/20140512193354/http
 
 On OS X with Homebrew, try [re]installing pkg-config:
 
+On Linux(you are awesome), try
```

Comment:
I think we should avoid comments suggesting some sort of favoritism. This is a library that's open for use on all the supported platforms. I suggest you remove the parentheses and the contents within them.

Additionally, this isn't Linux-specific, but Ubuntu-specific, so this shouldn be reflected in the line. I also wonder if this is the case for all Debian-derivative distributions. Could you check in a virtual machine if the same happens for Debian Jessie, for instance?